cfme/fixtures/vm_name.py

```diff
@@ -13,7 +13,7 @@ def vm_name(provider):
     yield vm_name
     try:
         logger.info('Cleaning up VM %s on provider %s', vm_name, provider.key)
-        provider.mgmt.delete_vm(vm_name)
+        provider.mgmt.delete_vm("{}0001".format(vm_name))
```

Comment:
I think this is too specific to service catalog provisioning requests for such a generic fixture.  Should check for existence of the '0001' vm

TODO above the fixture states cleanup shouldn't even happen here, but I don't quite agree with that.